sphinx/user_guide/reporters.rst

```diff
@@ -47,21 +47,21 @@ metrics. Set it to `train` or `test` to rely on the data provided to the constru
 addition, set `data_source` to `X_y` to pass a new dataset using the parameters `X` and
 `y`. This is useful when you want to compare different models on a new left-out dataset.
 
-While there are individual methods to compute each metric specific to the problem at
-hand, we provide the :class:`EstimatorReport.metrics.summarize` method that
-aggregates metrics in a single dataframe. By default, a set of metrics is computed based
-on the type of target variable (e.g. classification or regression). Nevertheless, you
-can specify the metrics you want to compute thanks to the `scoring` parameter. We accept
-different types: (i) some strings that correspond to scikit-learn scorer names or a
-built-in `skore` metric name, (ii) a callable or a (iii) scikit-learn scorer constructed
-with :func:`sklearn.metrics.make_scorer`.
+There are individual methods to compute each metric specific to the problem at hand. They return usual python objects such as floats, integers, or dictionaries.
 
 The second type of methods provided by :obj:`EstimatorReport.metrics` are methods that
 return a `skore` display object. They have a common API as well. They expose two
 methods: (i) `plot` that plots graphically the information contained in the display and
```

Comment:
Let's anticipate the PR of @sylvaincom to not have to come back here: we can mention the third method `.frame` that return a dataframe to programmatically access to the raw data.